.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
All the files inside the `.next/` folder are not needed to be part of this repository so I would first use this `.gitignore` file [https://github.com/akiran/nextjs-demo/blob/master/.gitignore](https://github.com/akiran/nextjs-demo/blob/master/.gitignore) and also I would remove these files with another commit as follows:
```bash
git reset --soft HEAD^ 
git reset HEAD .next/
git commit -c ORIG_HEAD  
git push -f
```